files/tagtacular/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "tagtacular",
+  "repo": "burnsbert/tagtacular",
+  "files": {
+    "include": ["tagtacular.js", "tagtacular.min.js", "tagtacular.css"]
```

Comment:
These files dont exist here https://github.com/burnsbert/tagtacular/tree/1.0.1
The bot will fail